changelog/1233.bugfix.rst

```diff
@@ -0,0 +1 @@
+Remove nmba-scipy from dependencies.
```

Comment:
```suggestion
Remove `numba-scipy` from dependencies.
```

I'd call this breaking instead of bugfix since you're deleting functionality from the package.

Reply:
I expect that this won't get intersphinxed so I'd suggest ``` ``numba-scipy`` ``` instead.  (It took all of ten back ticks to write that code snippet.)

Reply:
That, or keep it as is so if `numba-scipy` is added to the intersphinx mappings then it will start working.  ... Doesn't matter in the long run.